### PR TITLE
First update for 2020

### PR DESCRIPTION
**Pull request by william-liebenberg**
updated dates
updated venue
link to new prospectus
update logo to 2020
comment out old agenda
comment out old speakers
comment out old sponsors
update cfp section
update top-nav to hide speakers, show cfp
minor style tweak for mobile

new look:
serverless-homepage-2020

current look:
serverless-homepage-2019

### CODE REVIEW

File: src/site/config.toml [hunk 1]
```diff
@@ -6,38 +6,38 @@ title = "ServerlessDays Melbourne"
     logo = "/images/serverless_logo_melb.png"
     description = "A community based conference focusing on Serverless based platforms and technologies. Melbourne, August 29, 2019"
```
Comment (arthurscheeren) [inline src/site/config.toml]: We should also change it in here the 2019 description to "August 20th, 2020" ?

**Review by arthurscheeren — APPROVED** [review]

File: src/site/config.toml (hunk 1 above)
Comment (william-liebenberg) [reply src/site/config.toml]: Fixed :)

**Review by Jtango18 — APPROVED** [review]

**Review by mattgillard — COMMENTED** [review]
* remove ticket price
* Early bird tickets on sale (March 14) - maybe we can soft launch a week earlier

Other than that - looks good!

**Review by ajhooker82 — APPROVED** [review]
Looks good Will/Arthur. Good stuff. Just needs the updated links for ticketing, but thats to come of course.

**Pull request merged**